### PR TITLE
Describe installation with Symfony Flex

### PR DESCRIPTION
**Pull request by Zales0123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,27 +18,18 @@ Customer Order Cancellation Plugin that allows canceling the unpaid order straig
...
+
+> This installation instruction assumes that you're using Symfony Flex. If you don't, take a look at the
+[legacy installation instruction](docs/legacy_installation.md). However, we strongly encourage you to use
+Symfony Flex, it's much quicker! :)
+
+To install plugin, just require it with composer:
+
+```bash
+composer require sylius/customer-order-cancellation-plugin
+```
```
Comment (bartoszpietrzak1994) [inline README.md:31]: I'd also add this comment present in RefundPlugin's readme

```
Remember to allow community recipes with composer config extra.symfony.allow-contrib true or during plugin installation process
```

Comment (Zales0123) [reply README.md:31]: done

Comment (lchrusciel) [reply README.md:31]: Isn't this also true for https://github.com/Sylius/AdminOrderCreationPlugin/pull/96?

Comment (Zales0123) [reply README.md:31]: and also done 😄

**Review by bartoszpietrzak1994 — APPROVED** [review]

**Pull request merged**